src/applications/vre/28-1900/config/chapters/additional-information/additionalInformation.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  buildAddressSchema,
+  addressUISchema,
+} from '../../../../../disability-benefits/686c-674/config/address-schema';
+import { EDUCATION_LEVELS } from '../../constants';
+
+const newAddress = buildAddressSchema(true);
+// reset boolean type
+newAddress.properties['view:livesOnMilitaryBase'] = {
+  type: 'boolean',
+};
```

Comment:
This is necessary for the time being, since address schema is pulled from the new 686c schema...there will be a ticket that makes that schema form-agnostic.

Reply:
Lets make this ticket this sprint or next sprint